assets/js/app.js

```diff
@@ -21,12 +20,13 @@ import 'phoenix_html';
 
 // import socket from "./socket"
 
-import './header';
-import './filter_players';
-import './filter_trade_form';
-import './confirm_submit';
+import "./header";
+import "./filter_players";
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)